demo/component/BarChart.tsx

```diff
@@ -126,40 +127,36 @@ const pageData = [
   { name: 'Page G', uv: 189, pv: 4800, amt: 2400 },
 ];
 
-const RenderLabel = (props: any) => {
```

Comment:
I would suggest to keep the diff in a fix PR as small as possible. This seems to be an unrelated cleanup.

Reply:
yeah sorry I couldn't help myself - I can't stand all the errors in these demo files LOL